test/spec/ol/overlay.test.js

```diff
@@ -91,8 +91,7 @@ describe('ol.Overlay', function() {
         position: [0, 0]
       });
       map.addOverlay(overlay);
-      expect(overlay.element_.style.display).to.be('none');
```

Comment:
Unsure why you removed this test

Reply:
Previously, we expected the initial `display` to be `'none'` because we expected the map not yet to be rendered. But sometimes it was already rendered. So now, instead, we ensure the map is rendered, and we expect the `display` initially to be not `'none'`.

Reply:
makes sense